### ISSUE TITLE
Pointer lock controls error unlocking

### ISSUE THREAD

**Issue by torinmb** [mrdoob/three.js#20329]
##### Description of the problem

[Pointer lock controls](https://threejs.org/examples/?q=pointer#misc_controls_pointerlock) on the examples page throw an error after activating pointer lock, deactivating pointer lock, and reactivating pinter lock. This can cause the camera to also jump around to a random location.

To reproduce:
1) Click on the text saying 'Click To Play'
2) Press 'escape' on your keyboard to unlock the controls
3) Click on any part of the page other than 'Click To Play'

`
THREE.PointerLockControls: Unable to use Pointer Lock API
`

##### Three.js version
- [X ] r120
##### Browser
- [x] All of them

**Comment by Mugen87** [comment]
This seems to be a Chrome issue. Firefox and Safari work as expected.

The following (non-three.js) Pointer Lock demo shows that the second lock does only work after some attempts similar to the `three.js` demo. https://mdn.github.io/dom-examples/pointer-lock/

Do you mind reporting this to the Chrome bug tracker: https://bugs.chromium.org/p/chromium/issues/list

**Comment by torinmb** [comment]
@Mugen87 thanks for catching the bug in chrome. I submitted the bug here https://bugs.chromium.org/p/chromium/issues/detail?id=1127920. Hopefully, we'll hear back soon.

Did Arodic's solution end up fixing the camera position jump?

**Comment by Mugen87** [comment]
> Did Arodic's solution end up fixing the camera position jump?

Yes. 🎉

**Comment by Mugen87** [comment]
This is now the leading issue at the Chromium bug tracker:

https://bugs.chromium.org/p/chromium/issues/detail?id=1127223

Since the current Chromium implementation is quite annoying for users, I would appreciate if everyone could upvote/star the issue in order to give it a higher priority 😇 .

**Comment by inear** [comment]
Stumbled on to this issue as well. Calling the lock/unlock api directly does not seem to result in this error, only when using the 'esc' key to exit (and quickly trying to activate it again). So for now I will add a clear UI button to return to fps-camera and hope users forget that "press esc to show cursor" message. And also hide the reactivate-button for some seconds if I detect a esc-keypress when leaving fps-camera-mode.

**Comment by Mugen87** [comment]
> So for now I will hide the reactivate-button for some seconds if I detect a esc-keypress when leaving fps-camera-mode.

Unfortunately, this workaround does not always work. Think of a game where players quickly want to change between in-game and menu view. The current behavior in Chrome more or less breaks such use cases.

**Comment by inear** [comment]
True. Hopefully they will fix it. But meanwhile, could you use a different key to activate the menu and call unlock() yourself? Maybe not an option for you though.The error is not triggered if using the api directly, only if exit pointerlock with esc-button.
